chef/cookbooks/swift/templates/default/container-server.conf.erb

```diff
@@ -11,7 +11,6 @@ user = <%= node[:swift][:user] %>
 #
 # Use an integer to override the number of pre-forked processes that will
 # accept connections.
-# workers = auto
 workers = 2
```

Comment:
Makes sense to do this also in the parametrized values?

I think that is this only one of the fews conf files in swift that reflect the default value, but makes sense for me.